DotNetOnion.Tests/MonohopCircuits.cs

```diff
@@ -20,7 +20,7 @@ public class MonohopCircuits
         public async Task CanCreateMonohopCircuit()
         {
             using TorGuard guard = await TorGuard.NewClientAsync(torServer);
-            using TorCircuit circuit = new(guard);
```

Comment:
pls include stackoverflow URLs of what you showed me yesterday

Reply:
(in the commit message)

Reply:
Done